stacks/index.ts

```diff
@@ -1,9 +1,13 @@
-import { Tags } from 'aws-cdk-lib'
+import { Tags, RemovalPolicy } from 'aws-cdk-lib'
 import { PickupStack } from './PickupStack'
 import { BasicApiStack } from './BasicApiStack'
 import { App } from '@serverless-stack/resources'
 
 export default function (app: App): void {
+  if (app.stage !== "prod") {
+    app.setDefaultRemovalPolicy(RemovalPolicy.DESTROY)
```

Comment:
it's been useful to have `staging` not auto destroy things so we can see when we need to do a migration... the deploy will fail if the db needs a migration for example... 

as i understand it, sst destroys stateless resources like lambdas automatically by default when it needs to replace them, but will not do so for buckets and tables unless this removalPolicy default is changed.

https://docs.sst.dev/advanced/removal-policy#retained-resources

Reply:
I added this block as we have quite a few PR environments which have left-over dynamo, s3, etc related resources. Would it make sense to slightly modify the condition to:
```javascript
  if (!(app.stage in ['prod', 'staging'])) {
    app.setDefaultRemovalPolicy(RemovalPolicy.DESTROY)
  }
```